public/services/resolves/get-ip.js

```diff
@@ -30,12 +30,12 @@ export function getIp(
     const wazuhConfig = new WazuhConfig();
     const configuration = wazuhConfig.getConfig();
     const indexPatternFound = indexPatterns.find((indexPattern) => indexPattern.attributes.title === configuration.pattern);
-    if(!indexPatternFound){
-      AppState.removeCurrentPattern()
+    const currentPattern = AppState.getCurrentPattern();
+    if(!indexPatternFound && !indexPatterns.some((pattern) => pattern === currentPattern)){
+      AppState.removeCurrentPattern();
+      AppState.setCurrentPattern(indexPatternFound.id);
     }
```

Comment:
Test :x:
When change the index pattern using the Settings > Configuration section. After clink on the `Execute health check` buttons that appears when saving the configuration, the next screen is displayed:
![image](https://user-images.githubusercontent.com/34042064/115371508-52d3a580-a1ca-11eb-99be-ad4816bc5d74.png)

@pablomarga suggests which the next line could be causing the problem
```
AppState.setCurrentPattern(indexPatternFound.id);
```

There 2 things that I am not understand:
- To get in to the conditional a condition is `indexPatternFound` variable is `falsy`. Why is using `indexPatternFound.id` here if `indexPatternFound` is not a object?
- The comparation of `!indexPatters.some((pattern => pattern === currentPattern))` is comparing a object `pattern` againts a posible `string`.